tests/ui/rfc-2632-const-trait-impl/match-non-const-eq.stock.stderr

```diff
@@ -0,0 +1,13 @@
+error[E0015]: cannot match on `str` in constant functions
+  --> $DIR/match-non-const-eq.rs:6:9
+   |
+LL |         "a" => (),
+   |         ^^^
+   |
+   = note: `str` cannot be compared in compile-time, and therefore cannot be used in `match`es
+   = note: calls in constant functions are limited to constant functions, tuple structs and tuple variants
+   = help: add `#![feature(const_trait_impl)]` to the crate attributes to enable
```

Comment:
Does this help note appear to users on stable to? It seems confusing either way, as adding it doesn't fix the error (just makes it slightly more detailed).

Reply:
No it doesn't. Stable users don't get the suggestion to add unstable featured.